File: Arcade/Intro/javascript/sumUpNumbers.js
function sumUpNumbers(inputString) {
    let sum = 0, aux = "";

    for(let i in inputString) {
        if(/\D/.test(inputString[i])) {
            if(!isNaN(parseInt(aux))) sum += parseInt(aux);
            aux = "";
            i -= 1;
            continue;
        }
        aux += inputString[i];
    }
    
    if(!isNaN(parseInt(aux))) sum += parseInt(aux);
    
    return sum;
}

File: Arcade/Intro/Java/SumUpNumbers.java
final class SumUpNumbers {

	int sumUpNumbers(String inputString) {    
	    return Arrays.stream(inputString.replaceAll("[^\\d]", " ").split(" "))
	    .mapToInt(e -> !e.isEmpty() ? Integer.parseInt(e) : 0).sum();
	}
}